cli/config_file.rs

```diff
@@ -227,6 +227,56 @@ pub fn json_merge(a: &mut Value, b: &Value) {
   }
 }
 
+/// Based on an optional command line import map path and an optional
+/// configuration file, return a resolved module specifier to an import map.
+pub fn resolve_import_map_specifier(
+  maybe_import_map_path: Option<&str>,
+  maybe_config_file: Option<&ConfigFile>,
+) -> Result<Option<ModuleSpecifier>, AnyError> {
+  if let Some(import_map_path) = maybe_import_map_path {
+    if let Some(config_file) = &maybe_config_file {
+      if config_file.to_import_map_path().is_some() {
+        log::warn!("{} the configuration file \"{}\" contains an entry for \"importMap\" that is being ignored.", crate::colors::yellow("Warning"), config_file.specifier);
```

Comment:
If someone explicitly provides an import map as a cli flag I'm not sure we need to warn them about it because it should hopefully be obvious. Maybe we should warn though... not sure.

Reply:
I think it's fine, users can always specify `--quiet` to silence these warnings

Reply:
Because we have auto detection now, I think it is important, as you can be there scratching your head why your code isn't working. As noted `--quiet` can suppress it.